internal/backend/db/dbgorm/ownership_permissive_checker_test.go

```diff
@@ -5,12 +5,14 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.uber.org/zap"
 	"gorm.io/gorm"
 )
 
 func preOwnershipTestPermissive(t *testing.T) *dbFixture {
 	f := preOwnershipTest(t)
 	f.gormdb.owner = &PermissiveOwnershipChecker{f.gormdb.z}
+	f.gormdb.z.Info("patched ownership checker", zap.String("type", "permissive"))
```

Comment:
Why "patched"?

Reply:
gorm will be initialized by default with user checker. Tests are not using the same infrastructure as gormkitteh, thus I cannot pass which checker to use. So just printing in logs that we are patching to make it clear when you read test logs
```
{"level":"info","msg":"ownership","checker":"users"}
2024/07/16 16:05:01 OK   20240704121927_baseline.sql (3.9ms)
2024/07/16 16:05:01 OK   20240714051159_no-db-user.sql (2.85ms)
2024/07/16 16:05:01 goose: successfully migrated database to version: 20240714051159
=== RUN   TestSetVarWithOwnershipP
{"level":"info","msg":"patched ownership checker","type":"permissive"}
```